dtlstransport.go

```diff
@@ -31,10 +31,9 @@ type DTLSTransport struct {
 	certificates      []Certificate
 	remoteParameters  DTLSParameters
 	remoteCertificate []byte
-	// State     DTLSTransportState
+	state             DTLSTransportState
```

Comment:
Gonna be so great to have this in the api. Much faster to know if the connection has closed or not.